engine/flink/components/dev-table/src/main/scala/pl/touk/nussknacker/engine/flink/table/aggregate/Aggregator.scala

```diff
@@ -0,0 +1,29 @@
+package pl.touk.nussknacker.engine.flink.table.aggregate
+
+import pl.touk.nussknacker.engine.api.typed.typing.{Typed, TypingResult}
+
+// TODO: add remaining aggregators
+object Aggregator {
+  val allAggregators: List[Aggregator] = List(Sum, First)
```

Comment:
We have `enumeratum`, I think that we can use it

Reply:
Done :+1: